### PR TITLE
RIA-3793 Manage fee update for the type appeal not valid

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-3793](https://tools.hmcts.net/jira/browse/RIA-3793)

### Change description ###
Case-officer and Admin-officer manageFeeUpdate for the type Appeal not valid

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by mkayad — APPROVED** [review]
LGTM

**Review by makowalski — DISMISSED** [review]
LGTM

File: e2e/features/manage-fee-update.feature [hunk 1]
```diff
@@ -119,21 +119,131 @@ Feature: Manage fee update
...
 
     When I switch to be a `Judge`
     Then I click the `Appeal` tab
     And I should see the text `Fee update details`
     And I should see `Decision type changed` for the `Reason for fee update` field
-    And I should see `£10.00` for the `New fee amount (£)` field
+    And I should see `£10.00` for the `New fee amount` field
+    And I should see `Fee update recorded` for the `Completed stages` field
+
+  @Case-officer-manage-fee-update-appeal-not-valid @RIA-3793
```
Comment (divisathyan) [inline e2e/features/manage-fee-update.feature:139]: The e2e test should check if midevent is called and error is displayed for 'Appeal not valid' and fee amount >0
Please confirm

**Review by divisathyan — COMMENTED** [review]
Please see comment and confirm if it is correct

File: e2e/features/manage-fee-update.feature (hunk 1 above)
Comment (tarun-palisetty) [reply e2e/features/manage-fee-update.feature:139]: Updated the tests for £0

**Review by divisathyan — APPROVED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Pull request merged**